main.cc

```diff
@@ -25,9 +25,24 @@ int main(int argc, char* argv[]) {
             return 1;
         }
 
+        
+        // TODO: replace this with info from the config parser
```

Comment:
Awesome! I see where I can add my parsing of the config file now. Thanks for leaving a TODO here, Jordan! 👍